src/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.scss';
+import { AppContainer } from 'react-hot-loader';
+import App from './App';
+import registerServiceWorker from './registerServiceWorker';
+
+ReactDOM.render(<AppContainer>
+    <App />
+</AppContainer>, document.getElementById('root'));
+registerServiceWorker();
```

Comment:
I'm not sure if we need this in our sandbox app. Do you guys want to learn this?